diff -up ./plugin/Makefile.in.orig ./plugin/Makefile.in
--- ./plugin/Makefile.in.orig	2013-08-18 18:28:42.780912728 +0000
+++ ./plugin/Makefile.in	2013-08-18 18:29:02.021134922 +0000
@@ -292,7 +292,7 @@ gretlzip.la: gretlzip.lo $(ZIPOBJ)
 	$(LINK) -o $@ $^ $(ZIP_LIBS) $(GRETLLIB)
 
 audio.la: audio.lo midi_utils.lo
-	$(LINK) -o $@ $^ $(GRETLLIB) $(AUDIO_LIBS)
+	$(LINK) -o $@ $^ $(GRETLLIB) $(AUDIO_LIBS) $(GTK_LIBS)
 
 odbc_import.la: odbc_import.lo
 	$(LINK) -o $@ $^ $(GRETLLIB) $(ODBC_LIBS)
